autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -354,6 +377,34 @@ def llm_call(
                 response.get("prompt_eval_count") or 0,
                 response.get("eval_count") or 0,
             )
+        elif provider == "open_router":
+            client = openai.OpenAI(
+                base_url="https://openrouter.ai/api/v1",
+                api_key=credentials.api_key.get_secret_value(),
+            )
+
+            response = client.chat.completions.create(
+                extra_headers={
+                    "HTTP-Referer": "https://agpt.co",
```

Comment:
Do we want these always being agpt/AutoGPT, even for self host?

Reply:
Yeah, it's just for a usage based leaderboard for our application on their website so the more "points" the better, and there's no expectation to distinguish them.